src/cadence/scripts/getNFTs.cdc

```diff
@@ -401,16 +403,53 @@ pub fun getEverbloom(owner: PublicAccount, id: UInt64): NFTData? {
     )
 }
 
+// https://flow-view-source.com/mainnet/account/0xc38aea683c0c4d38/contract/Eternal
```

Comment:
Could you share a link to the testnet as well?

Reply:
https://github.com/alchemyplatform/alchemy-flow-contracts/pull/16 @niveda-krish

Reply:
Thank you!